.github/workflows/ci.yaml

```diff
@@ -69,4 +69,4 @@ jobs:
           flags: unittests
           file: cover.out
       - name: Check diff
-        run: '[[ -z $(git status -s) ]] || (printf "Existing modified/untracked files.\nPlease run \"make generate manifests\" and push again.\n"; exit 1)'
+        run: 'pwd || [[ -z $(git status -s) ]] || (printf "Existing modified/untracked files.\nPlease run \"make generate manifests\" and push again.\n"; exit 1)'
```

Comment:
I should explain that when I did a commit, there was an error before for ` [[ -z $(git status -s) ]]` is false. Then I make a trick to make it true always.